tests/server/api/types/test_Dataset.py

```diff
@@ -7,6 +8,13 @@
 from strawberry.relay import GlobalID
 
 
+class DatasetFixture(NamedTuple):
```

Comment:
Curious what makes this necessary?

Reply:
So the ids are not hardcoded but are returned from the the fixture. That way more than one fixture than be used at the same time.

Reply:
it's not totally necessary. I can also revert.

Reply:
It doesn't seem like we want to be using more than one fixture at a time after all.

Reply:
removed

Reply:
I think I probably agree about removing the hard-coded IDs, but the IDs that get created are deterministic if the hard-coded ID is removed (starting at 1 and increasing by one each time), so you can still hard-code them in the test case. Not sure this data structure makes it easier to read the test IMO, but non-blocking.

Reply:
I think it's fair to use more than one fixture at a time. We do it in a couple of tests.

Reply:
Since the ID's will be set by the ordering of the fixtures if more than one are used, or if the fixtures are session scoped, setting it by hand is probably not going to be convenient.